src/process/slices/transientSlice.js

```diff
@@ -8,10 +8,13 @@ const transientSlice = createSlice({
   reducers: {
     updateProps: (state, action) => {
       return { ...state, ...action.payload };
+    },
+    resetState: () => {
```

Comment:
This is the above i was saying.

Reply:
resetState should be defined before updateProps

Reply:
done